draft-aesedepece-pos.md

```diff
@@ -0,0 +1,766 @@
+criteria:
+
+1. Select the block candidate whose proposer has the highest power,
+2. If 2 or more block candidates were proposed by stakers with the same power, select the block candidate whose VRF 
+   output has the lowest big-endian value, and
+3. If 2 or more block candidates were proposed by stakers with the same power, and every of those block candidates  
+   have the same VRF output, select the one whose block hash has the lowest big-endian value.
+
+**()** Upon consolidating a block candidate, the amount of coins in the proposer's stake entry in the `StakesTracker` 
+MUST be incremented by an amount equivalent to the block reward, plus the addition of all unclaimed value from the 
+transactions contained in the block, i.e. the miner fees.
+
+**()** Upon consolidating a block candidate, the average age of the proposer's stake entry in the `StakesTracker` MUST
+be reset to zero, i.e. the average epoch MUST be set to the same epoch as the one for which the block was consolidated.
+
+#### Witnessing eligibility
+
+**()** For any given data request transaction in the commitment stage, a replication factor `rf` MUST be computed as
+the `witnesses` field in the data request multiplied by 2 to the power of the current witnessing round:
+
+`rf = witnesses * 2 ^ round`
+
+**()** For a commitment transaction to be considered for inclusion in a block, it MUST hold an amount of power and
+provide a valid VRF proof that MUST satisfy all the following requirements:
+1. the VRF input is the concatenation of the previous block hash, the data request hash, and the current epoch, all 
+   of them in big-endian format,
+2. the power of the block proposer is in the `rf / stakers`th quantile among the powers of all the stakers, and
+3. the big-endian value of the VRF output is less than `max_rounds * own_power / (max_power * (rf - max_rounds) - rf * 
+   threshold)`,
+
+Where:
+- `rf` is the replication factor,
+- `stakers` is the number of stakers in the `StakesTracker`,
+- `max_rounds` is the maximum number of commitment rounds allowed by the protocol (currently `4`),
+- `own_power` is the power of the committer,
+- `max_power` is the power of the most powerful staker, and
+- `threshold` is the power of the least powerful staker within the `rf/stakers`th quantile from requirement 2 above.
+
+**()** May a node receive and successfully validate a number of concurrent commitments with distinct transaction hashes
+for the same data request and epoch greater than the value of the `witnesses` field in the data request, it SHOULD 
+prioritize for inclusion in an eventual block candidate those commitments whose VRF output have the lowest big-endian 
+value.
+
+**()** Upon consolidating a block candidate, for each of the commitment transactions contained therein, the amount of
+coins in each committer's stake entry in the `StakesTracker` MUST be decremented by an amount equivalent to the 
```

Comment:
This statement does not align with the latest simulation results I shared and on which this description is based. You have to reset all coins of the selected stakers to age 1. Otherwise the most powerful stakers wull always be eligibile to solve a data request.